internal/subscription/core.go

```diff
@@ -69,6 +69,14 @@ func (c *Core) CreateSubscription(ctx context.Context, m *Model) error {
 		return merror.New(merror.NotFound, "project not found")
 	}
 
+	if ok, err = c.topicCore.ExistsWithName(ctx, m.Topic); err != nil {
```

Comment:
woudn't get call below will do same, if topic name not found ?

Reply:
CreateTopic used to return a "key not found" error. Have added an exists check in the GetTopic function itself now which would return "topic not found" in case of a wrong input.